src/OV/OpenVarioMenu.cpp

```diff
@@ -99,6 +99,115 @@ FileMenuWidget::Prepare([[maybe_unused]] ContainerWindow &parent,
   });
 }
 
+class ScreenTimeoutWidget final
+  : public RowFormWidget
+{
+  UI::Display &display;
+  UI::EventQueue &event_queue;
+
+public:
+  ScreenTimeoutWidget(UI::Display &_display, UI::EventQueue &_event_queue,
+                 const DialogLook &look) noexcept
+    :RowFormWidget(look),
+     display(_display), event_queue(_event_queue) {}
+
+private:
+  /* virtual methods from class Widget */
+  void Prepare(ContainerWindow &parent,
+               const PixelRect &rc) noexcept override;
+  void SaveTimeout(int timeoutvalue);
+};
+
+void
+ScreenTimeoutWidget::SaveTimeout(int timeoutInt)
+{
+   ChangeConfigInt("timeout", timeoutInt, "/boot/config.uEnv");
+}
+
+void
+ScreenTimeoutWidget::Prepare([[maybe_unused]] ContainerWindow &parent,
+                             [[maybe_unused]] const PixelRect &rc) noexcept
+
+{
+  AddButton("immediately", [this](){
+    SaveTimeout(0);
+        static constexpr const char *argv[] = {
+      "/bin/sh", "-c", 
+      "echo Automatic timeout was set to 0s", 
+      nullptr
+    };
```

Comment:
WTF, you wrote a shell script and all it does is display a message to be displayed by the menu???